v2/pom.xml

```diff
@@ -658,6 +658,7 @@
         <module>datastream-to-sql</module>
         <module>elasticsearch-common</module>
         <module>file-format-conversion</module>
+        <module>googleads-to-bigquery</module>
```

Comment:
For scalability reasons, we are keeping new google product to google product templates under `googlecloud-to-googlecloud` module so please move this template to `googlecloud-to-googlecloud` module as well.